.pylintrc

```diff
@@ -28,4 +28,29 @@ ignored-classes=numpy,scipy,autograd,qiskit
 # can either give multiple identifiers separated by comma (,) or put this option
 # multiple times (only on the command line, not in the configuration file where
 # it should appear only once).
-disable=line-too-long,invalid-name,too-many-lines,redefined-builtin,too-many-locals,duplicate-code,bad-continuation
+disable=
+  line-too-long,
+  invalid-name,
+  too-many-lines,
+  redefined-builtin,
+  too-many-locals,
+  duplicate-code,
+  cyclic-import,
+  import-error,
+  bad-option-value,
+  import-outside-toplevel,
+  missing-class-docstring,
+  missing-function-docstring,
+  no-self-use,
+  no-member, # because of qnode decorator
+  comparison-with-callable,
+  unsubscriptable-object, # because of qnode decorator
+  not-callable, # because of qnode decorator
+  unexpected-keyword-arg,
+  arguments-differ,
+  no-value-for-parameter
```

Comment:
Seems that PL ignores these in the tests but not in the source?

Reply:
Fixed this!